erpnext_com/www/docs/user/manual/en/CRM/appointment.md

```diff
@@ -0,0 +1,58 @@
+# Appointment
+
+```
+---
+title: Appointment
+add_breadcrumbs: 1
+show_sidebar: 0
+
+metatags:
+ description: Appointment Schedulling for CRM in ERPNext
+ keywords: Appointment Schedulling , CRM, frappe, erpnext new features, erp, open source erp, free erp, security
+---
+```
+
+**An appointment is a prearranged meeting between a lead and an employee.**
+
+Appointment document type can be used to schedule and manage interaction with a lead or opportunity. 
```

Comment:
Hi, the definition looks very generic and not ERPNext specific, you can maybe merge both the first and the second statements and see if it works...check for grammatical errors, minor changes of a missed comma here and there are required. Do let me know if you need any help.